ui/espresso/ScreenshotSample/app/build.gradle

```diff
@@ -31,6 +31,23 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    testOptions {
+        devices {
+            // run with ../gradlew  -Pandroid.sdk.channel=3 -Pandroid.experimental.androidTest.useUnifiedTestPlatform=true nexusOneApi30DebugAndroidTest
+            nexusOneApi30 (com.android.build.api.dsl.ManagedVirtualDevice) {
+                // A lower resolution device is used here for better performance, although using
+                // an ATD image will greatly reduce the performance impact of emulating higher
+                // resolution screens
```

Comment:
This description is a bit confusing. You say you use a lower resolution device for better performance. But in the same sentence you say the impact of higher vs lower resolution is minimal when using ATD. So why are you still using a lower resolution device?